docs/azure_arc_jumpstart/azure_arc_k8s/aks_hybrid/aks_edge_essentials_full/_index.md

```diff
@@ -0,0 +1,238 @@
+---
+type: docs
+title: "AKS Edge Essentials full deployment"
+linkTitle: "AKS Edge Essentials full deployment"
+weight: 2
+description: >
+---
+
+## Create an AKS Edge Essentials full deployment with two VMs in Hyper-V nested virtualization in an Azure Windows Server VM, and connect the Hyper-V VMs and AKS Edge Essentials cluster to Azure Arc using Azure Bicep
+
+The following Jumpstart scenario will show how to Create an AKS Edge Essentials full deployment with two VMs in Hyper-V nested virtualization in an Azure Windows Server VM, and connect the Hyper-V VMs and AKS Edge Essentials cluster to Azure Arc using [Azure Bicep](https://learn.microsoft.com/azure/azure-resource-manager/bicep_template/overview). The provided Azure Bicep templates are responsible for creating the Azure resources as well as executing the LogonScript (AKS Edge Essentials cluster creation and Azure Arc onboarding (Hyper-V VMs and AKS Edge Essentials cluster)) on the Azure Windows Server VM.
+
+> **NOTE: It is not expected to use a nested virtualization in a production environment, let alone using an Azure VM to do so. The below scenario is unsupported and should ONLY be used for demo and testing purposes.**
+
+![Architecture diagram](./01.png)
```

Comment:
you need to take a screenshot when PowerPoint is in presentation mode. you can't have these red wave lines